packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -1036,33 +1035,36 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
     const { isOpen } = this.state;
     const disabled = this._isDisabled();
 
-    if (!disabled && !(this._isOpenedByKeyboardFocus && isOpen)) {
+    if (!disabled && !this._shouldOpenOnFocus()) {
```

Comment:
```suggestion
    // `shouldOpenOnFocus` is used here because we don't want to toggle the `isOpen` state if we're supposed to open the Dropdown on focus.
    if (!disabled && !this._shouldOpenOnFocus()) {
```

Reply:
i will update these in my future PR :)